SPEAKING.md

```diff
@@ -0,0 +1,29 @@
+# Speaking at VanJS
+
+A typical VanJS event has two presentation slots, each between 20 and 45 minutes long. Each slot may be either filled by a feature talk, or by a set of 2-4 lightning talks of roughly 5 minutes each.
+
+Who speaks at VanJS? **You do!** Here are the guidelines we use to guide what talks we schedule, so you can consider what you’d like to present on.
+
+*As of December, 2018*
+
+- **Speaker Lead Generator:** Jeremy Gonzalez
+- **Presentation and Topic Coordinator:** Shawn McKay
+
+## Topics
+- Ideally a feature presentation tells some kind of story. Often it has a beginning, middle, and an end – stereotypically that would be a problem, some lessons, resulting in a new way of looking at things.
+- New and exciting features and APIs released by browsers and popular JS frameworks are perennial popular topics.
+- Ambitious projects and demos also make good talks. If your project is getting the browser to do something that is rather hard, or people wouldn’t expect to work in a browser, there’s often a good presentation to be had there.
+- Hard-won lessons make good talks as well. If your team tried something, and it was hard – whether it succeeded or failed – you likely learned something you could share.
+- Topic variation is a key theme of VanJS. There are React meetups and CSS meetups and Ember meetups, but at VanJS we have a chance to show different topics from month to month, and pair up topics on different technologies, and feature some “softer” topics about software engineering more generally. 
+
+## Advanced vs. Beginner Content
+- It’s more enjoyable to see a talk that blows your mind and inspires you to learn more, than it is to see a talk that’s boring because you already know the content. With that in mind, we encourage presenters to aim for talks with intermediate or senior developers in mind.
+- Even though VanJS content is most often aimed at intermediate developers, VanJS is an inclusive event and we get lots of bright and aspiring new developers attending. Speakers should not condescend new developers or people coming from different development backgrounds.
+
+## Speakers
+- VanJS has not done a good enough job on speaker diversity historically, so in our outreach it’s worth putting in extra effort to contact potential speakers who may be from underrepresented groups in tech. More diverse speakers bring out more diverse audiences, which is good for the group and Vancouver’s tech community as a whole.
```

Comment:
Making a suggestion that we switch from language describing our intent to what we're actually doing :)

I also put in a line for people who want reasons for supporting diversity.

```suggestion
- VanJS has not done a good enough job on speaker diversity historically, so we are putting in extra effort to contact potential speakers who may be from underrepresented groups in tech. Diverse talent adds strength and resilience to the creative process. More diverse speakers bring out more diverse audiences, which is good for the group and Vancouver’s tech community as a whole.
```